coney_island.gemspec

```diff
@@ -21,7 +21,7 @@ Gem::Specification.new do |s|
 
   s.add_dependency "rails", ">= 4.0.1"
   s.add_dependency "amqp", ">= 1.5.0"
-  s.add_dependency "bunny", ">= 1.7.0"
+  s.add_dependency "bunny", "< 2.0"
```

Comment:
We are locking it down so it *doesn't* upgrade, the 2.0 version was causing random channel errors.